cypress/integration/taskGenerateTableDom.spec.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+Cypress.Commands.add('tableInputs', (fvalue, svalue, value) => {
+  cy.contains(fvalue).then(($fvalue) => {
+    cy.contains(svalue).then(($svalue) => {
+      cy.contains(value).then(($value) => {
+        const child1 = $fvalue.get(0).offsetLeft;
+        const child2 = $svalue.get(0).offsetLeft;
+        const main = $value.get(0).offsetLeft;
+
+        expect(main).to.equal(child1);
+        expect(main).to.equal(child2);
+      });
+    });
+  });
+});
+
+describe('Genetate table app', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should contain a first column as a Name', () => {
```

Comment:
should we test by columns or rows?

Reply:
some students create tables in desc order, so I decided to test columns